python/common/org/python/types/NoneType.java

```diff
@@ -6,6 +6,167 @@
 
     NoneType() {}
 
+    @org.python.Method(
+        __doc__=""
+    )
+    public org.python.Object __pos__() {
+        throw new org.python.exceptions.TypeError("bad operand type for unary +: 'NoneType'");
+    }
+
+    @org.python.Method(
+        __doc__=""
+    )
+    public org.python.Object __neg__() {
+        throw new org.python.exceptions.TypeError("bad operand type for unary -: 'NoneType'");
+    }
+
+    @org.python.Method(
+        __doc__=""
+    )
+    public org.python.Object __invert__() {
+        throw new org.python.exceptions.TypeError("bad operand type for unary ~: 'NoneType'");
+    }
+
+
+    @org.python.Method(
+        __doc__=""
+    )
+    public org.python.Object __bool__() {
+        return new org.python.types.Bool(false);
+    }
+
+    @org.python.Method(
+        __doc__ = ""
+    )
+    public org.python.Object __ge__(org.python.Object other) {
+        if(other instanceof org.python.types.Bool) {
```

Comment:
Minor style point - if isn't a function call, so there should be a space after each if.